### ISSUE TITLE
Name conflicts not handled correctly?

### ISSUE THREAD

**Issue by VertexMachine** [hugoattal/OrbitalMarket#61]
When I click on maximise button on my asset pack it gets redirected to someone's else asset pack. See attached video:

https://github.com/user-attachments/assets/4becfd34-2004-47c5-8405-f175e4fb8f7f

Steps to reproduce:
1) Search for "low poly nature", find my (vertex rage studio) asset pack (at time of writing it was 1st in ranking for default filters)
2) Click on it, click to maximise view

Result: redirected to wrong asset
Expected result: should redirect to correct asset page

**Comment by hugoattal** [comment]
Yep, I have a small bug on slugs with product with the same name 😕... It's gonna be fixed in the upcoming updates!

**Comment by hugoattal** [comment]
(It's fixed)
